### DISCUSSION TITLE
Find Feature

### DISCUSSION THREAD

**Discussion by mbierman** [isontheline/pro.webssh.net#707] [General]
I am almost sure I asked this before and you answered it... but I can't remember _and_ I can't find the answer so, forgive me I'm asking again. 

How can I find text that was printed in an open terminal e.g. previous commands (Yeah, I know I can use history for that) or the output of commands.

**Comment by isontheline** [comment]
I'm not sure you have already asked 

Are you talking about the ability to search a command with "bash" Ctrl-R?

Or about the ability to search on the buffer? => #539

Search on buffer is on my main roadmap within 30/45 days.

**Comment by mbierman** [comment]
Hi @isontheline, I was thinking of the buffer (past commands and outputs). I'm very excited about this enhancement as I do this all the time!
